### DISCUSSION TITLE
Snap&Run

### DISCUSSION THREAD

**Discussion by Averagedude69420** [3kh0/ext-remover#1073] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
My school has the snap read extension but I don't know what useful scripts can be used with it can yall give me a list of useful scripts?

**Comment by AshtonDavies** [comment, marked as answer]
**Here are some examples:**
[Edpuzzle Answers](https://github.com/ading2210/edpuzzle-answers)
[Blooket Hacks](https://github.com/Glixerz/Blooket-Hacks/tree/main) | [bot/non-bookmarklet version](https://schoolcheats.net/blooket)
[Kahoot Hack](https://github.com/futzumi/Kahoot-Hack/blob/main/bookmarklet.js)


_I have not tested anything in the list above, but you get the idea now._

[Escape characters](https://www.w3schools.com/tags/ref_urlencode.ASP) are necessary for the Snap&Read extension to load the link content properly in case you didn't know.

**Reply by Averagedude69420** [reply]
How do you make the script load properly in Snap&Read?

**Reply by Blobby-Boi** [reply]
the best thing I found is to just use a bookmarklet that runs bookmarklets (This should work I think)
javascript:(function() {     var code = prompt("Enter bookmarklet code:");     if (code) {         try {             eval(unescape(code));         } catch (error) {             alert("Error executing code: " + error);         }     } else {         alert("No code entered.");     } })();

**Ashton's Edit:**
`javascript:%21function%28%29%7Bvar%20code%3Dprompt%28%22Enter%20bookmarklet%20code%3A%22%29%3Bif%28code%29try%7Beval%28unescape%28code%29%29%7Dcatch%28e%29%7Balert%28%22Error%20executing%20code%3A%20%22%2Be%29%7Delse%20alert%28%22No%20code%20entered.%22%29%7D%28%29%3B`
_Use the above bookmarklet for quick execution in Snap&Read._

**Reply by AshtonDavies** [reply]
> the best thing I found is to just use a bookmarklet that runs bookmarklets (This should work I think)
> javascript:(function() {     var code = prompt("Enter bookmarklet code:");     if (code) {         try {             eval(unescape(code));         } catch (error) {             alert("Error executing code: " + error);         }     } else {         alert("No code entered.");     } })();

Oh, I keep forgetting about that.

**Comment by hankypoo7** [comment]
does this not work anymore?

**Reply by AshtonDavies** [reply]
It needs fixing

**Reply by hankypoo7** [reply]
what kind? I'm a little bit of a skid but can you work on this if you have time because I need to run bookmarklets for most other exploits

**Comment by Charlieis1nsan3** [comment]
how do I even use God damn snap&Run I cant find anything that helps!!!

**Reply by hankypoo7** [reply]
rs
